content/en/infrastructure/resource_catalog/_index.md

```diff
@@ -142,6 +142,10 @@ Use the **Recent Changes** tab on a resource's side panel to see a 7-day history
 
 See the [AWS Config integration page][6] to launch a CloudFormation template that sets up change event forwarding through AWS Config.
 
+#### Azure
+
+After confirming that you have enabled resource collection under the **Resource Collection** tab in the [Azure integration tile][14], submit your request using the [form above](#recent-changes). There are no other setup steps.
```

Comment:
@andresromerodatadog Does the wording here look ok? 

@buraizu Maybe recommend switching to something like this - 
"After you've submitted your request to the form above, please ensure Resource Collection is turned on for the Azure subscriptions in the [Azure Integration tile][14] to also collect resource configuration changes."